inf-clojure.el

```diff
@@ -577,7 +577,7 @@ The prefix argument SWITCH-TO-REPL controls whether to switch to REPL after the
 If you are using REPL types, it will pickup the most approapriate
 `inf-clojure-var-doc-form` variant."
   (pcase inf-clojure-repl-type
-    (lumo inf-clojure-var-doc-form-lumo)
```

Comment:
I tried that backtick in `ielm` and it did not work, it was always going to the else branch. Also in `cider` I have never seen using symbols so I settled for strings.